server/Controller.js

```diff
@@ -0,0 +1,50 @@
+const { addItem,
+    deleteItem,
+    getItem,
+    updateItem } = require('./Services')
+
+const getItems = async (req, res) => {
+
+
+    const data = await getItem();
+    console.log(data)
+    res.send(data)
+
+
+}
+
+const addItems = async (req, res) => {
```

Comment:
There should be no business logic in controllers.

Reply:
only services are being called here

Reply:
If else condition means some logic, we should not have any sorts of check in controller even if it's just calling different service methods, this should have been handled in service only.

What could we have done here is, we should have called service `add_item`, in that method on basis of params we should have called different model methods as per our requirement.

Reply:
okay ! got it , I was not aware of that